test/mocks/init/mocks.h

```diff
@@ -50,6 +50,23 @@ class ExpectableTargetImpl : public TargetImpl {
   ::testing::internal::TypedExpectation<void()>& expectInitializeWillCallReady();
 };
 
+class ExpectableEagerTargetImpl : public EagerTargetImpl {
```

Comment:
Can the code duplication here be refactored out?

Reply:
`ExpectableEagerTargetImpl` doesn't need `fn` and this mock is not helpful. will remove